### ISSUE TITLE
console: union types don't get included in remote schema permissions

### ISSUE THREAD

**Issue by abhi40308** [hasura/graphql-engine#6779]
Remote schema permissions do not support adding union types.

GraphQL SDL:

```
type Launch {
    id: ID!
    site: String
    mission: Mission
    rocket: Rocket
    isBooked: Boolean!
  }
  type Rocket {
    id: ID!
    name: String
    type: String
  }
union launchRocket = Launch | Rocket
```

Type `launchRocket` does not show up in UI:

<img width="660" alt="Screenshot 2021-04-06 at 5 13 55 PM" src="https://user-images.githubusercontent.com/26903230/113705781-793e0f00-96fb-11eb-8072-298af16e7150.png">

Introspection query result

<img width="407" alt="Screenshot 2021-04-06 at 5 19 14 PM" src="https://user-images.githubusercontent.com/26903230/113706407-46e0e180-96fc-11eb-8998-cddcf94f5993.png">

**Comment by jgoux** [comment]
It seems to be fixed in v2.0.0-beta.1 👍

**Comment by abhi40308** [comment]
Addressed in https://github.com/hasura/graphql-engine/commit/f24caef627a6d566158f7fa3a85aae5bfb422389
